roles/sensu-check/handlers/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 - name: restart sensu-client missing ok
   service: name=sensu-client state=restarted must_exist=false
+  failed_when: false
```

Comment:
it's because common role failed at meta: flush handlers, since sensu-client is not installed yet, and even have must_exist=false, still failed.
Now I remove the notify sensu-client restart from logging role's monitoring.yml (it's run before common role),  and it works now.